node/tests/cli.rs

```diff
@@ -115,6 +115,7 @@ fn read_config_override_values() {
     let options = RunOptions {
         storage_backend: Some(backend_type.clone()),
         node_type: Some(node_type),
+        mock_time: None,
```

Comment:
No test for this?

Reply:
I'm not sure what to test here. `mock_time` is only present in `RunOptions`.